fixtures/build.js

```diff
@@ -22,28 +20,32 @@ window.EXAMPLE_BUILD = {
       }
     },
     "strategy": {
-      "type": "STI",
-      "stiStrategy": {
-        "builderImage": "openshift/ruby-20-centos7:latest",
-        "image": "openshift/ruby-20-centos7:latest"
+      "type": "Source",
+      "sourceStrategy": {
+        "from": {
+          "kind": "DockerImage",
```

Comment:
achievement unlocked: soul crusher

it's mostly correct, just missing the "DockerImage" Kind.  Does that help?